sormas-api/src/main/java/de/symeda/sormas/api/feature/FeatureType.java

```diff
@@ -63,6 +63,16 @@
 		new FeatureType[] {
 			CASE_SURVEILANCE }),
 
+	// SHOW/HIDE VIEW TAB FEATURES
+	VIEW_TAB_CASES_HOSPITALIZATION(false, true, null),
```

Comment:
For all of the new feature types, set `serverFeature` to `true` and `dependentFeatures` to `new FeatureType[] { CASE_SURVEILANCE }` or `new FeatureType[] { CONTACT_TRACING }` respectively